src/transformer/utils/get_user.php

```diff
@@ -19,6 +19,11 @@
 
 function get_user(array $config, \stdClass $user) {
     $fullname = get_full_name($user);
+
+    if (is_enabled_config($config, 'send_jisc_data')) {
+        $fullname = $user->username;
+    }
+
```

Comment:
as discussed, this is not required.
It is covered by #745 / #746

Reply:
I will remove it.